azure/05-app-services/mongoHelpers.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright 2016-2019, Pulumi Corporation.  All rights reserved.
+
+import * as pulumi from "@pulumi/pulumi";
+
+export function parseConnString(
+    conns: pulumi.Output<string[]>,
+): pulumi.Output<{ [key: string]: string }> {
+    // Per the official docs[1], the format of this connection string is:
+    //
+    //   mongodb://username:password@host:port/[database]?ssl=true
+    //
+    // Where these could have the following values:
+    //
+    //   {
+    //     username: "cosmosdb93a4133a",
+    //     password: "23maXrWsrzZ1LmPe4w6XNGRJJTHsqGZPDTjyVQNbPaw119KCoCNpStH0DQms5MKdyAecisBM9uWbpV7lUnyNeQ==",
+    //     host: "cosmosdb93a4133a.documents.azure.com",
+    //     port: "10255",
+    //     database: "mydatabase"
+    //   }
+    //
+    // There are a few subtleties involved in getting the Bitnami node Chart to actually be able to
```

Comment:
Can we show off a quick example of the bitnami mongo chart being deployed created in `index.ts` that consumes the `mongoConnStrings` k8s Secret?

Reply:
Is this the app-services or the apps stack?

Reply:
Apps stack please